### PR TITLE
[OSCD]win_pe_exec_vsjitdebugger.yml added

### PR DESCRIPTION
**Pull request by esebese**

### CODE REVIEW

**Comment by zinint** [comment]
Hello, @esebese, welcome and thanks for your contribution, I'll check the rule and will get back to you!

**Comment by zinint** [comment]
image
Everything seems fine, thanks for your work!

**Comment by zinint** [comment]
I think it's ready to be merged! @yugoslavskiy

File: rules/windows/process_creation/win_pe_exec_vsjitdebugger.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+title: PE File Execution via Vsjitdebugger
+id: 4b51f73f-1583-4202-a8e0-2d4bbf4beeee
+status: experimental
+description: Detects the execution of Vsjitdebugger tool as parent process which is utilized like proxy for other PE files executions.
+references:
+    - https://lolbas-project.github.io/lolbas/OtherMSBinaries/Vsjitdebugger/
+author: 'Ensar Şamil, @sblmsrsn, OSCD Community'
```
Comment (yugoslavskiy) [inline rules/windows/process_creation/win_pe_exec_vsjitdebugger.yml:7]: Hello @esebese! Could you please change `OSCD Community` to the `oscd.community` or `@oscd_initiative` just like you did for the other rules? thank you!

Comment (zinint) [reply rules/windows/process_creation/win_pe_exec_vsjitdebugger.yml:7]: Thanks for mentioning this detail.

@esebese - just a `ping`, when you have time, please make the changes.

**Comment by zinint** [comment]
@esebese - just a ping, when you have time, please make the changes.

File: rules/windows/process_creation/win_pe_exec_vsjitdebugger.yml (hunk 1 above)
Comment (esebese) [reply rules/windows/process_creation/win_pe_exec_vsjitdebugger.yml]: I edited.

Comment (yugoslavskiy) [reply rules/windows/process_creation/win_pe_exec_vsjitdebugger.yml]: Thanks!

**Comment by yugoslavskiy** [comment]
Thank you very much, @esebese! Well done!

**Review by yugoslavskiy — APPROVED** [review]
@thomaspatzke @Neo23x0 guys, I believe that this PR could be merged (:
please let us know what do you think about it if there is anything you would like to change.

**Pull request merged**